.papr.yml

```diff
@@ -6,13 +6,13 @@ branches:
     - try
 
 required: true
-context: f29-primary
+context: f30-primary
 
 # This test case wants an "unprivileged container with bubblewrap",
 # which we don't have right now; so just provision a VM and do a
 # docker --privileged run.
 host:
-  distro: fedora/29/atomic
+  distro: fedora/30/atomic
```

Comment:
This doesn't exist.  Also I don't think you need to bump it, just the container right?

Reply:
I don't know. Which container?

Reply:
Oh, I see. You're saying run an F30 container on F29. Yeah that should work.